specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/Dataset.json

```diff
@@ -533,6 +622,57 @@
         "location"
       ]
     },
+    "ExcelDataset": {
+      "x-ms-discriminator-value": "Excel",
+      "description": "Excel dataset.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Dataset"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "description": "Excel dataset properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ExcelDatasetTypeProperties"
+        }
+      }
+    },
+    "ExcelDatasetTypeProperties": {
+      "description": "Excel dataset properties.",
+      "type": "object",
+      "properties": {
+        "location": {
+          "$ref": "#/definitions/DatasetLocation",
+          "description": "The location of the excel storage."
+        },
+        "sheetName": {
+          "type": "object",
+          "description": "The sheet of excel file. Type: string (or Expression with resultType string)."
+        },
+        "range": {
+          "type": "object",
+          "description": "The partial data of one sheet. Type: string (or Expression with resultType string)."
+        },
+        "firstRowAsHeader": {
+          "type": "object",
+          "description": "When used as input, treat the first row of data as headers. When used as output,write the headers into the output as the first row of data. The default value is false. Type: boolean (or Expression with resultType boolean)."
```

Comment:
All of these should be `string` and not `object`. firstRowHeader should be `boolean`